packages/core/test/Synchronizer.ts

```diff
@@ -0,0 +1,198 @@
+// @ts-ignore
+import { ethers } from 'hardhat'
+import { BigNumberish, BigNumber } from 'ethers'
+import { expect } from 'chai'
+import { formatProofForSnarkjsVerification } from '@unirep/circuits'
+import * as config from '@unirep/circuits'
+import { genEpochKey, schema } from '@unirep/core'
```

Comment:
the `schema` should be import from `../src` so that it will contain the `'Post'`, `'Comment'`...

Reply:
importing the right schema fixed the database the previous error`error processing event: Error: anondb error: Error: anondb error: Error: Unable to find table Post`.